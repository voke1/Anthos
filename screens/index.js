// ## Cart
import MyCart from "./Cart/myCart";
import Gallery from "./Gallery/gallery";
import Home from "./Home.js/home";

// ## Admin
import AddProduct from "./Admin/addProduct";

export { MyCart, Gallery, Home, AddProduct };
