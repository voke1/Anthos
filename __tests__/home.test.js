import React from "react";
import { render, fireEvent } from "@testing-library/react-native";
import { Home } from "../screens/";
import renderer from "react-test-renderer";


// Snapshots test
it("Should render home component", async () => {
  const renderedComponent = renderer.create(<Home />).toJSON();
expect(renderedComponent).toMatchSnapshot();

});
