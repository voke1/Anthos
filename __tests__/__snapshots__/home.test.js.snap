// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render home component 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    accessibilityIgnoresInvertColors={true}
    style={
      Object {
        "alignItems": "center",
        "borderRadius": 33.35,
        "flex": 1,
        "justifyContent": "flex-start",
        "paddingVertical": 33.35,
      }
    }
  >
    <Image
      source={1}
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "height": undefined,
            "width": undefined,
          },
          undefined,
        ]
      }
    />
    <View
      style={
        Object {
          "flex": 1,
          "justifyContent": "space-between",
        }
      }
    >
      <View
        style={
          Object {
            "justifyContent": "space-around",
            "marginVertical": 100.05000000000001,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#000000",
              "fontFamily": "Poppins-Regular",
              "fontSize": 31,
              "lineHeight": 28,
              "paddingHorizontal": 13.34,
              "textAlign": "center",
            }
          }
        >
          Various ornamental plants are here
        </Text>
        <Text
          style={
            Object {
              "color": "#000000",
              "fontFamily": "Poppins-Regular",
              "fontSize": 13,
              "lineHeight": 18,
              "paddingHorizontal": 13.34,
              "paddingVertical": 66.7,
              "textAlign": "center",
            }
          }
        >
          find the variety of ornamental plants that you like here. We have many plants that you can choose at will.
        </Text>
        <View
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "alignSelf": "center",
              "backgroundColor": "#61C7A4",
              "borderRadius": 60.029999999999994,
              "elevation": 5,
              "height": 120.05999999999999,
              "justifyContent": "center",
              "marginBottom": 13.34,
              "marginHorizontal": 160.07999999999998,
              "opacity": 1,
              "width": 120.05999999999999,
            }
          }
        >
          <Image
            resizeMode="contain"
            source={1}
            style={
              Object {
                "elevation": 5,
                "height": 15,
                "tintColor": "#fff",
                "width": 15,
              }
            }
          />
        </View>
      </View>
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#61C7A4",
            "borderRadius": 13.34,
            "elevation": 5,
            "height": 80.03999999999999,
            "justifyContent": "center",
            "marginBottom": 13.34,
            "marginHorizontal": 160.07999999999998,
            "opacity": 1,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
              "fontFamily": "Poppins-Bold",
              "fontSize": 14,
              "fontWeight": "bold",
              "lineHeight": 21,
            }
          }
        >
          Admin
        </Text>
      </View>
    </View>
  </View>
</RCTSafeAreaView>
`;
